RHIDP-1493: separate plugin title for RHDH (#199)
redhat-developer/red-hat-developers-documentation-rhdh: titles/plugin-rhdh/title-plugin.adoc

[id='title-plugin']
include::artifacts/attributes.adoc[]
= Configuring plugins in {product}
:context: plugin-rhdh
:doctype: book
:imagesdir: images

> The {product} is an enterprise-grade, open developer platform that you can use to build developer portals. This platform contains a supported and opinionated framework using various plugins that helps reduce the friction and frustration of developers while boosting their productivity.

Reviewer: ```suggestion
The {product} is an enterprise-grade, integrated developer platform, extended through plugins, that helps reduce the friction and frustration of developers while boosting their productivity.
```

Author: Done
